tests/integration-tests/class-test-case.php

```diff
@@ -1,32 +1,27 @@
 <?php
 
-namespace Beans\BeansFramework\Tests;
+namespace Beans\BeansFramework\Tests\IntegrationTests;
 
 use Brain\Monkey;
-use PHPUnit\Framework\TestCase;
+use WP_UnitTestCase;
 
-abstract class Beans_Test_Case extends TestCase {
+/**
+ * Abstract Class Test_Case
+ *
+ * @package Beans\BeansFramework\Tests
+ */
+abstract class Test_Case extends WP_UnitTestCase {
 
 	/**
 	 * Prepares the test environment before each test.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @return void
 	 */
 	protected function setUp() {
 		parent::setUp();
 		Monkey\setUp();
```

Comment:
It's possible that we don't need to mock anything WP-specific in integration tests, since WP will be available, so either Mockery or mocks in PHPUit could be used.

In which case, this whole class would not be needed, and integration tests could extend `WP_UnitTestCase` directly.

Reply:
@GaryJones It's possible.  Once we start writing those tests, then we can adjust this Test Case, if needed.